static/docs/commands-reference/status.md

```diff
@@ -38,7 +38,10 @@ stages that affect the target stage.
 
 In the `local` mode, changes are detected through the checksum of every file
 listed in every stage file in the pipeline against the corresponding file in the
-file system. The output indicates the detected changes, if any. If no
+file system, using two modes `changed checksum` when actual checksum of data
```

Comment:
these are not modes. These are different outputs/statuses. The paragraph that starts with "If instead differences ..." should be updated. Check the list of different outputs that starts line 57-60.